File: client/src/app/shared/models/operator-permission.ts
import {T001} from './t001';

export class OperatorPermission {
  id: string;
  defaultReceiveT001: T001;
  allT001s: boolean;
  t001s: T001[];

  static assign(...sources: any[]): OperatorPermission {
    const result = Object.assign(new OperatorPermission(), ...sources);
    return result;
  }
}

File: server/src/jpaModelgen/java/com/hengyi/japp/cargo/domain/OperatorPermission_.java
package com.hengyi.japp.cargo.domain;

import com.hengyi.japp.cargo.domain.sap.T001;
import javax.annotation.Generated;
import javax.persistence.metamodel.SetAttribute;
import javax.persistence.metamodel.SingularAttribute;
import javax.persistence.metamodel.StaticMetamodel;

@Generated(value = "org.hibernate.jpamodelgen.JPAMetaModelEntityProcessor")
@StaticMetamodel(OperatorPermission.class)
public abstract class OperatorPermission_ {

	public static volatile SingularAttribute<OperatorPermission, T001> defaultReceiveT001;
	public static volatile SingularAttribute<OperatorPermission, Boolean> allT001s;
	public static volatile SingularAttribute<OperatorPermission, String> id;
	public static volatile SingularAttribute<OperatorPermission, Operator> operator;
	public static volatile SetAttribute<OperatorPermission, T001> t001s;

}

